### PR TITLE
use crun instead of runc

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Switching to crun, as crun is the defacto reference implementation of OCI for cgroup v2.

### CODE REVIEW

**Comment by giuseppe** [comment]
@AkihiroSuda have you a chance to check the crun git master version?

**Comment by AkihiroSuda** [comment]
Attempted to use github artifact binary from master, but couldn't wget/curl 😢 

Will try again later.

**Comment by AkihiroSuda** [comment]
Restarted CI with https://github.com/AkihiroSuda/crun/releases/tag/snapshot-c69ea8c

File: boot/crio.sh [hunk 1]
```diff
@@ -11,6 +11,16 @@ mkdir -p $XDG_DATA_HOME/containers/oci/hooks.d $XDG_CONFIG_HOME/containers $XDG_
 if [[ ! -f $XDG_CONFIG_HOME/crio/crio.conf ]]; then
 	cat >$XDG_CONFIG_HOME/crio/crio.conf <<EOF
 registries = ['registry.access.redhat.com', 'registry.fedoraproject.org', 'docker.io']
+[crio.runtime]
+  default_runtime = "crun"
+  [crio.runtime.runtimes]
+    [crio.runtime.runtimes.crun]
+      runtime_path = "$U7S_BASE_DIR/bin/crun"
+      runtime_root = "$XDG_RUNTIME_DIR/crio/crun"
+# override built-in "runc" config to avoid error: 'runtime validation: \"runc\" not found in $PATH'
```
Comment (AkihiroSuda) [inline boot/crio.sh]: @giuseppe Do you know the solution for this? Setting `default_runtime = "crun"` doesn't solve.

Comment (AkihiroSuda) [reply boot/crio.sh]: Opened https://github.com/cri-o/cri-o/issues/3360 for asking

**Pull request merged**